server/utils/utils.test.ts

```diff
@@ -45,3 +45,40 @@ describe('withDefaultTime', () => {
     expect(withDefaultTime(undefined)).toEqual(undefined)
   })
 })
+
+describe('gzipBase64AndEncode', () => {
+  it('should return a base64 encoded gzipped string', () => {
+    const input = 'Hello, world!'
+    const result = gzipBase64AndEncode(input)
+    expect(result.substring(0, 12)).toEqual('H4sIAAAAAAAA')
```

Comment:
had a problem with the 13th character - was `E` locally and `A` when running in circle so have split out.  Both versions decoded to the same text 🤷